functionnal_tests/cypress/integration/search/search_contents_spec.js

```diff
@@ -53,22 +53,22 @@ describe('Searching keywords', () => {
       cy.get(contentHtmlDocGetter).should('be.not.visible')
     })
 
-    describe('then archiving one document', () => {
-      describe('and searching the same keyword and validating again', () => {
-        it('Should not display the archived document', () => {
-          cy.get(searchInput).type(threadTitle).type('{enter}')
+    // describe('then archiving one document', () => {
```

Comment:
add the comment rules of algoo ("INFO - GB - data") and some explanation about why you are commenting this code with a reference to an issue or to something else